### ISSUE TITLE
Environment value are not correctly updated

### ISSUE THREAD

**Issue by Dudesons** [jonnydgreen/terraform-provider-postman#17]
### Terraform Version
```
Terraform v1.3.9
```

### Affected Resource(s)
Please list the resources as a list, for example:
- `postman_environment`

### Terraform Configuration Files
```hcl
resource "postman_environment" "cohort" {
  name      = var.project
  workspace = local.workspace_id

  values = [
    {
      key   = "firebaseApiKeyWallet"
      value = data.google_firebase_web_app_config.app.api_key
      type  = "secret"
    },
    {
      key   = "firebaseApiKeyAdmin"
      value = data.google_firebase_web_app_config.admin.api_key
      type  = "secret"
    },
    {
      key   = "adminUrl"
      value = "https://admin-api.${var.root_domain}"
      type  = "default"
    },
 ]
}
```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
When a value change this value is applied on the current value

### Actual Behavior
When an environment variable is updated the current value is not correctly updated, only the initial value is updated.
In order to bypass that I need to remove the content of the env

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. write code for creating an environment
2. `terraform apply`
3. edit some values of the environment
4. `terraform apply`
5. check postman

**Comment by jonnydgreen** [comment]
Hi, thanks for submitting this issue! Would you be interested in submitting a pull request to fix this?

**Comment by Dudesons** [comment]
> Hi, thanks for submitting this issue! Would you be interested in submitting a pull request to fix this?

Yes, I can try the only things I didn't find is the api documentation for postman.
There is a documentation available or you are looking for network calls ?

**Comment by jonnydgreen** [comment]
Awesome! 🚀 (apologies for the late reply, hectic day yesterday!)

Good question, the documentation for the postman API is here: https://www.postman.com/postman/workspace/postman-public-workspace/api/72a32ca3-f06a-4e83-a933-2821a0e6616f/definition/024624cf-e622-4836-9d19-efd067f6cbdb/file/024624cf-e622-4836-9d19-efd067f6cbdb?version=860f56fe-6024-4403-a5ae-1b95532ba7b9

And you can find the sanitised version that used in the repo here: https://github.com/jonnydgreen/terraform-provider-postman/blob/main/openapi.yaml

More details around contributing can be found here: https://github.com/jonnydgreen/terraform-provider-postman/blob/main/CONTRIBUTING.md

If you have any further questions, don't hesitate to ask! :)
